### DISCUSSION TITLE
Import webhooks

### DISCUSSION THREAD

**Discussion by nbaju1** [jfrog/terraform-provider-xray#171] [Q&A] [answered]
Simple question: Is it possible to import webhooks? If yes, what is the correct syntax for specifying the id of the webhook to import?

**Comment by alexhung** [comment, marked as answer]
@nbaju1 Good question! Yes import is supported but I noticed the import example is missing in the documentation, which I will fix.

To import the webhook, the command should be:

```console
terraform import xray_webhook.my_webhook WebhookName
```
